src/storage/sstable.rs

```diff
@@ -231,7 +241,13 @@ where
         let mut sparse_index = SparseIndex::new();
         let mut offset = 0 as FileOffset;
 
-        for (kv_i, res_kv) in Self::merge_range(tables, &None, &None)?.enumerate() {
+        for (kv_i, res_kv) in Self::merge_range(
+            tables,
+            None::<&Box<dyn Fn(&K) -> Ordering>>,
+            None::<&Box<dyn Fn(&K) -> Ordering>>,
```

Comment:
There is an unstable feature that will eventually allow us to shorten this to `None::<!>`.